### DISCUSSION TITLE
root3/Support App guidance

### DISCUSSION THREAD

**Discussion by mrrobertbuss** [community/community#136402] [New to GitHub]
Hello everyone: 
I have installed this app and cannot get 4 Tiles to work. 1. Battery Level - shows nothing. 2. Storage - shows nothing. 3. Submit a Service Desk goes nowhere (mailto:servicedesk@stretto.com) 4. Privacy & Security link doesn't open. Any help with this is greatly appreciated.

**Comment by mrrobertbuss** [comment]
I was able to get 2 Tiles to work. 
Can someone send me the configuration for the Battery to show it's percentage within a Tile?
Also, can someone send me the configuration for Privacy & Security to open within a Tile? 
root3 support has suggested Battery is built in with the Info Items 1 though 4 and that is not true in my case. Instead I am using First Row Left for the Battery and First Row Right for Privacy & Security. screenshots.

I am stuck how to make these work and would appreciate any scripts to make this work.
![Screenshot 2024-08-26 at 9 42 25 AM](https://github.com/user-attachments/assets/6a462a8c-0cd9-402c-8729-f5ae85388bf1)
![Screenshot 2024-08-26 at 9 38 48 AM](https://github.com/user-attachments/assets/df561dcc-209a-4d1b-8617-a77d227f9b99)
![Screenshot 2024-08-26 at 9 31 59 AM](https://github.com/user-attachments/assets/7dda700d-ba46-402c-bc17-463ce76b0530)

**Comment by ghostinhershell** [comment]
Hi @mrrobertbuss, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

You may also want to try asking for help somewhere that focuses on this particular project or app. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
